scripts/lib/libvirt/admin-config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import argparse
+from string import Template
+import sys
+
+import libvirt
+import libvirt_helpers
+
+conn = libvirt.open("qemu:///system")
+if not conn:
+    print("Failed to open connection to the hypervisor")
+    sys.exit(1)
+
+parser = argparse.ArgumentParser(description="Create Admin Node Config.")
+parser.add_argument("cloud", type=str, help="Name of the cloud")
+parser.add_argument("adminnodememory", type=str, help="Amount of Memory")
+parser.add_argument("adminvcpus", type=str, help="Number of VCPUs")
+parser.add_argument("emulator", type=str, help="Emulator")
+parser.add_argument("adminnodedisk", type=str, help="Disk of the Admin Node")
+parser.add_argument("localrepositorymount", type=str,
+                       help="Local Repository Mount XML")
+
+args = parser.parse_args()
+
+
+def main():
+    cloud = args.cloud
```

Comment:
All this code should be in a reusable module function, e.g.

    def write_admin_node_xml(filename, values):
        ...

    def main():
        # parse args
        write_admin_node_xml("/tmp/%s-admin.xml" % args.cloud, vars(args))

This means that when the caller gets switched from bash to Python, it can simply call the module function directly, rather than forking a new process which has to do all the CLI handling stuff.